src/components/LanguageDropdown/LanguageDropdown.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ReactComponent as LanguagePicker } from './LanguagePicker.svg';
+import { languageOptions } from '../../data/LanguageContent';
+
+import './LanguageDropdown.css';
+
+const LanguageDropdown = ({ language, setLanguage }) => {
+  return (
+    <div className='languageDropdown'>
+      <div className='label'>
+        <LanguagePicker height='2rem' width='2rem' />
+        <h4>{ language.language }:</h4>
```

Comment:
Here, I would like to see more deconstruction like you did with the line 7 params so at line 8 I would add a new line of const { language = 'what ever the default is' } = language and this will also help with testing